kore/src/Kore/Logger/WarnMissingHook.hs

```diff
@@ -0,0 +1,68 @@
+{-|
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+-}
+
+module Kore.Logger.WarnMissingHook
+    ( WarnMissingHook (..)
+    , warnMissingHook
+    ) where
+
+import Data.Set
+    ( Set
+    )
+import Data.Text
+    ( Text
+    )
+import Data.Text.Prettyprint.Doc
+    ( (<+>)
+    )
+import qualified Data.Text.Prettyprint.Doc as Pretty
+import Data.Typeable
+    ( Typeable
+    )
+
+import Kore.Internal.Symbol
+    ( Symbol
+    )
+import Kore.Logger
+import Kore.Unparser
+    ( unparse
+    )
+
+
+data WarnMissingHook = WarnMissingHook
+    { hook :: Text
+    , symbol :: Symbol
+    } deriving (Eq, Typeable)
+
+_scope :: Scope
```

Comment:
I don't intend for this to have its own scope because we can always match on the actual warning constructor. Instead, the scope can be added by the caller if appropriate.